controllers/Users/userController.js

```diff
@@ -0,0 +1,39 @@
+const User = require('../../lib/modelManagers/usermodel');
+
+const pagination = require('../../lib/utils/pagination/paginationHelper');
+
+/**
+ *
+ *
+ * @class Products
+ */
+class UsersController {
+  /**
+   * @static
+   * @returns { object } object
+   * @param {*} req
+   * @param {*} res
+   */
+  static async listUsers(req, res) {
+    const { limit, offset } = pagination(req.query);
+    try {
+      const searchResult = User.listAllUsers(limit, offset);
+      const users = searchResult;
+      if (searchResult) {
+        res.status(200).send({
+          msg: 'users retrieved succesfully ',
+          users,
+        });
+      } else {
+        res.status(400).send({
+          msg: 'error retrieving users',
+        });
+      }
+    } catch (error) {
+      return error;
```

Comment:
It is always best to throw an unhandled error so that it can be handled properly by the caller

Reply:
fixed